adrs/5541831634/README.md

```diff
@@ -0,0 +1,160 @@
+* You **must** use `<fractalDir>/i18n/numbers` to store i18n number
+  formatting. Read more about the intent at [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat).
+* You **must** use `<fractalDir>/i18n/plural` to store i18n
+  pluralization formatting. Read more about the intent at [Intl.PluralRules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules).
+* You **must** use `<fractalDir>/i18n/relative-time` to store i18n
+  relative-time formatting. Read more about the intent at [Intl.RelativeTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat).
+* You **must** use `<fractalDir>/helpers/datetime` to store anything related
+  to Dates or Times. Except formatters, use `<fractalDir>/i18n/datetime`
+  instead.
+* You **must** use `<fractalDir>/helpers/currency` to store anything related
+  to currency. Except formatters, use `<fractalDir>/i18n/numbers`
+  instead.
+* You **must** use `<fractalDir>/helpers` to store anything that does not
+  fall into any of the other helper directory categories. You **may** create
+  your own files and directories based on your context.
+* You **must** use `<fractalDir>/transports/http` to store anything
+  related to HTTP transport clients.
+* You **must** use `<fractalDir>/transports/web-socket` to store
+  anything related to Web Socket transport clients.
+* You **must** use `<fractalDir>/transports/graphql` to store
+  anything related to GraphQL transport clients.
+* You **must** use `<fractalDir>/storages` to store anything related to the
+  device. You **may** use `local`, `session`, `cookies`, `websql`, and
+  `indexeddb` as subdirectories.
+* You **must** use `<fractalDir>/schemas` to store anything related to Joi, Yup
+  schemas or any similar package. You **may** use **joi**, **yup** or
+  package name as subdirectories.
+
+## Routes directory structure
+
+### Web
+
+* `<rootDir>/src/routes/index.tsx` **must** be used as the root path
+  of the URL.
+
+  ```text
+  URL: http://acme.io/
+  <rootDir>
+  └── src
+      └── routes
+          └── index.tsx
+  ```
+
+* URL subpath **must** must match a subdirectory inside `<fractalDir>/routes`.
+
+  ```text
+  URL: http://acme.io/authors/articles
+  <rootDir>
+  └── src
+      └── routes
+          ├── authors
+          │   ├── index.tsx
+          │   └── routes
+          │       └── articles
+          │           └── index.tsx
+          └── index.tsx
+  ```
+
+* Dynamic URL subpath **must** must match a subdirectory inside
+  `<fractalDir>/routes` using `[the dynamic section identifier]` syntax. In case
+  you use NextJS, you must use the same name as the dynamic NextJS route
+  directory.
+
+  ```text
+  URL: http://acme.io/authors/articles/:articleId/
+  <rootDir>
+  └── src
+      └── routes
```

Comment:
Where would I put static data like markdown content files?

Reply:
It depends on what you mean by static data.

1. Are those markdown file [mdx files](https://mdxjs.com/)?
  11. Do they behave like a page? Then use `routes`
  1.2. Are they reusable across the app? Then use `components`

2. Are those markdown statics and don't need to be processed? If so use the `public` directory